tools/python/FindOptimizerOpsetVersionUpdatesRequired.py

```diff
@@ -59,18 +62,53 @@ def get_call_args_from_file(filename, function_or_declaration):
     return results
 
 
+def get_multiline_call_args_from_file(filename, function_or_declaration):
+    """
+    Search a file for all function calls or declarations that match the provided name.
+    Allows the opening '(' and closing ')' to be split across multiple lines.
+    Supports a single call per line.
+    """
+
+    results = []
+    with open(filename) as f:
+        cur_line = None
+
+        for line in f:
+            if not cur_line:
+                # look for new match
+                start = line.find(function_or_declaration)
+                if start != -1:
+                    cur_line = line
```

Comment:
Does it make sense to store from where the function starts to save the need for `cur_line.find(function_or_declaration)`? This way we can do `cur_line[len(function_or_declaration): end]` and be rid of the pyright error `"find" is not a known member of "None"`

Reply:
Changed to store from the '(' so a `find` isn't needed when appending to results.